draft-ietf-webpush-protocol.xml

```diff
@@ -1500,7 +1500,7 @@ HEADERS      [stream 82] +END_STREAM
     <section title="Acknowledgements">
       <t>
         Significant technical input to this document has been provided by Ben
-        Bangert, Peter Beverloo, Kit Cambridge, JR Conlin, Lucas Jenns, Matthew Kaufman,

```

Comment:
Yeah, that's a problem.  We can't (yet) use non-ascii for these documents (which is super annoying).  What is your preferred ASCII form Jens or Jenss perhaps?